src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -155,6 +156,8 @@ internal class SerializableReferenceLocation
 
         public SerializableSymbolUsageInfo SymbolUsageInfo { get; set; }
 
+        public ImmutableArray<AdditionalProperty> AdditionalProperties{ get; set; }
+        public AdditionalProperty ContainingMemberInfo { get; set; }
```

Comment:
I presume you want to delete `public AdditionalProperty ContainingMemberInfo { get; set; }`?